project/vision_backend/management/commands/vb_inspect_extracted_features.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+from collections import defaultdict
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from images.models import Source, Image, Point
+from lib.utils import direct_s3_read
+
+from vision_backend.tasks import reset_features
+
+
+class Command(BaseCommand):
+    help = "Crawls extracted features and checks that they align with " \
+           "DB content. Optionally also correct them."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument('--do_correct',
+                            type=int,
+                            nargs='?',
+                            default=0,
+                            help="If true, fix erroneous images.")
+
+    def handle(self, *args, **options):
+
+        errors = defaultdict(list)
+        for source in Source.objects.filter():
+            print('Inspecting source: {}'.format(source.name))
```

Comment:
Unicode again. Either do the same fix as the other PR, or add `from __future__ import unicode_literals` at the top (the latter makes it a bit easier to port to Python 3).